content/app/development/logic/serveraction/_index.en.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Server Action
+linktitle: Server Action
+description: How to write custom server side actions that can be triggered by a API-client or a generic button
+toc: true
+---
+
+{{% panel theme="warning" %}}
+⚠️ Server actions require version 8.0.0 or newer of app-libs
+
+If you want to define a generic button version 4.0.0 or newer of app-frontend is required.
+{{% /panel %}}
+
+## Overview
+
+Server actions is a way to write custom backend code that end users can trigger either by pressing a button or making an api request.
+
+They are almost identical to process actions, the only difference is that they do not automatically change the process state when triggered.
+
+A server action is tied to one or more process tasks and needs explisit autorization rules to grant end users execution rights.
+
+## Create and configure server action
+
+### Configure available server action for a process task
+
+To register what server acitons are available for a process task we need to add it to the tasks config in the process.bpmn file
+
+Example of a process task with a server action called `myServerAction`
+
+```xml
+<!-- Beginning of process definition omitted for brevity -->
+        <bpmn:task id="Task_1" name="Utfylling">
+            <bpmn:incoming>SequenceFlow_1</bpmn:incoming>
+            <bpmn:outgoing>SequenceFlow_2</bpmn:outgoing>
+            <bpmn:extensionElements>
+                <altinn:taskExtension>
+                    <altinn:taskType>data</altinn:taskType>
+                    <altinn:actions>
+                        <altinn:action type="serverAction">myServerAction</altinn:action>
+                    </altinn:actions>
+                </altinn:taskExtension>
+            </bpmn:extensionElements>
+        </bpmn:task>
+<!-- End of process definition omitted for brevity -->
+```
+
+### Add access polic
+
+To allow users to trigger the server action we need to add authorization policies to the policy.xml file.
+
+Example of access policy rule granting users with `DAGL` role access to trigger the server action `myServerAction` when the process is in `Task_1`
```

Comment:
Consider adding a screen shot of this rule configured in Altinn Studio to show the alternative way of working with authorization rules.